CI failure due to traceback:

```
Traceback (most recent call last):
  File "/tmp/ansible_th8wkv4k/ansible_module_service_facts.py", line 217, in <module>
    main()
  File "/tmp/ansible_th8wkv4k/ansible_module_service_facts.py", line 202, in main
    svc = svcmod.gather_services()
  File "/tmp/ansible_th8wkv4k/ansible_module_service_facts.py", line 191, in gather_services
    services[service_name] = {"name": service_name, "state": state_val, "source": "systemd"}
UnboundLocalError: local variable 'service_name' referenced before assignment
```

--- a/lib/ansible/modules/system/service_facts.py
+++ b/lib/ansible/modules/system/service_facts.py
@@ -180,13 +180,12 @@ class SystemctlScanService(BaseService):
             return None
         rc, stdout, stderr = self.module.run_command("%s list-units --no-pager --type service --all" % systemctl_path, use_unsafe_shell=True)
         for line in [svc_line for svc_line in stdout.split('\n') if '.service' in svc_line and 'not-found' not in svc_line]:
+            service_name = line.split()[0]
             if "running" in line:
                 state_val = "running"
             else:
                 if 'failed' in line:
                     service_name = line.split()[1]
-                else:
-                    service_name = line.split()[0]
                 state_val = "stopped"
             services[service_name] = {"name": service_name, "state": state_val, "source": "systemd"}
         return services

